src/main/scala/viper/gobra/frontend/info/implementation/property/Convertibility.scala

```diff
@@ -19,9 +19,11 @@ trait Convertibility extends BaseProperty { this: TypeInfoImpl =>
     case (Single(lst), Single(rst)) => (lst, rst) match {
       case (left, right) if assignableTo(left, right) => true
       case (IntT(_), IntT(_)) => true
+      case (SliceT(IntT(config.typeBounds.Byte)), StringT) => true
```

Comment:
Is this case supported by the encoding? If yes, could you write a testcase that uses this case.

Reply:
I reported this in issue #241

Reply:
Does it hold in general that a slice of bytes can be assigned to a string (even without explicitly using `string(...)`)?
If not, it seems to me that it would make more sense to relax the inputs to the `string(...)` function

Reply:
In Go, a []byte can be converted to a string using `string(...)`. Without the explicit conversion, it leads to a type error. Gobra follows the same approach here.